From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Mon, 20 Feb 2023 07:07:52 +0000
Subject: [PATCH] Enable new password manager autofill UI by default

This instead uses a keyboard accessory on top of virutal keyboard to
avoid UX issues of drop-down suggestions obscuring other dialogs, such
as switching keyboard
---
 components/autofill/core/common/autofill_features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/autofill/core/common/autofill_features.cc b/components/autofill/core/common/autofill_features.cc
index 2246179592da9..db8549d0ea31a 100644
--- a/components/autofill/core/common/autofill_features.cc
+++ b/components/autofill/core/common/autofill_features.cc
@@ -611,7 +611,7 @@ BASE_FEATURE(kAutofillVirtualCardsOnTouchToFillAndroid,
 // instead of the regular popup.
 BASE_FEATURE(kAutofillKeyboardAccessory,
              "AutofillKeyboardAccessory",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Controls whether the Autofill manual fallback for Addresses and Payments is
 // present on Android.
